### PR TITLE
Update UI resources Japanese translations

### PR DESCRIPTION
**Pull request by yoshinorin**
Hi.

I updated Japanese translation.
I'm going to try to translate others step by step.

Thank you :)

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/core/CoreResources_ja.properties [hunk 1]
```diff
@@ -455,7 +455,7 @@ dialog_connection_edit_wizard_shell_cmd_directory_title =\u4F5C\u696D\u30C7\u30A
 dialog_connection_edit_wizard_shell_cmd_variables_hint_label =\u30B3\u30DE\u30F3\u30C9\u3067\u5909\u6570\u3092\u4F7F\u7528\u3059\u308B\u3053\u3068\u304C\u3067\u304D\u307E\u3059\u3002\u30AF\u30EA\u30C3\u30AF\u3059\u308B\u3068\u30EA\u30B9\u30C8\u304C\u8868\u793A\u3055\u308C\u307E\u3059\u3002
 dialog_connection_edit_wizard_shell_cmd_variables_hint_title =\u5909\u6570
 
-dialog_connection_edit_wizard_connections =\u30AF\u30E9\u30A4\u30A2\u30F3\u30C8ID
```
Comment (yoshinorin) [inline plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/core/CoreResources_ja.properties:458]: The current translation is a little confusing.

File: plugins/org.jkiss.dbeaver.ui.editors.data/OSGI-INF/l10n/bundle_ja.properties [hunk 2]
```diff
@@ -84,10 +102,10 @@ command.org.jkiss.dbeaver.core.resultset.toggleOrder.name=\u7D50\u679C\u306E\u30
 command.org.jkiss.dbeaver.core.resultset.toggleOrder.description=\u7D50\u679C\u306E\u30BD\u30D5\u30C8\u30AA\u30FC\u30C0\u30FC\u3092\u5207\u308A\u66FF\u3048\u308B\uFF08\u6607\u9806/\u964D\u9806/\u30C7\u30D5\u30A9\u30EB\u30C8\uFF09
 command.org.jkiss.dbeaver.core.resultset.filterSettings.name=\u30AB\u30B9\u30BF\u30E0\u30D5\u30A3\u30EB\u30BF ...
 command.org.jkiss.dbeaver.core.resultset.filterSettings.description=\u30AB\u30B9\u30BF\u30E0\u30D5\u30A3\u30EB\u30BF
-command.org.jkiss.dbeaver.core.resultset.filterSave.name=Save filter settings
+command.org.jkiss.dbeaver.core.resultset.filterSave.name=\u30D5\u30A3\u30EB\u30BF\u8A2D\u5B9A\u3092\u4FDD\u5B58
 command.org.jkiss.dbeaver.core.resultset.filterSave.description=\u73FE\u5728\u306E\u30AA\u30D6\u30B8\u30A7\u30AF\u30C8\u306E\u30D5\u30A3\u30EB\u30BF\u8A2D\u5B9A\u3092\u4FDD\u5B58\u3059\u308B
-command.org.jkiss.dbeaver.core.resultset.filterClear.name=\u3059\u3079\u3066\u306E\u30D5\u30A3\u30EB\u30BF/\u30AA\u30FC\u30C0\u30FC\u3092\u524A\u9664\u3059\u308B
-command.org.jkiss.dbeaver.core.resultset.filterClear.description=\u3059\u3079\u3066\u306E\u30D5\u30A3\u30EB\u30BF/\u30AA\u30FC\u30C0\u30FC\u3092\u524A\u9664\u3059\u308B
```
Comment (yoshinorin) [inline plugins/org.jkiss.dbeaver.ui.editors.data/OSGI-INF/l10n/bundle_ja.properties:90]: The current translation is a little confusing.

File: plugins/org.jkiss.dbeaver.ui.editors.sql/OSGI-INF/l10n/bundle_ja.properties [hunk 3]
```diff
@@ -135,13 +147,14 @@ colorDefinition.org.jkiss.dbeaver.sql.editor.color.disabled.background.descripti
...
 page.org.jkiss.dbeaver.preferences.main.sqleditor.name = SQL\u30A8\u30C7\u30A3\u30BF
 page.org.jkiss.dbeaver.preferences.main.sql.completion.name = SQL\u306E\u5B8C\u4E86/\u6298\u308A\u305F\u305F\u307F
+page.org.jkiss.dbeaver.preferences.main.sql.codeeditor.name = \u30B3\u30FC\u30C9\u30A8\u30C7\u30A3\u30BF
 page.org.jkiss.dbeaver.preferences.main.sql.format.name = SQL\u66F8\u5F0F\u8A2D\u5B9A
 page.org.jkiss.dbeaver.preferences.main.sql.dialects.name = SQL\u30C0\u30A4\u30A2\u30EC\u30AF\u30C8\u306E\u8A2D\u5B9A
 page.org.jkiss.dbeaver.preferences.main.sql.resources.name = \u30B9\u30AF\u30EA\u30D7\u30C8
 page.org.jkiss.dbeaver.preferences.main.sqlexecute.name = SQL\u51E6\u7406
 page.org.jkiss.dbeaver.preferences.main.sql.templates.name = \u30C6\u30F3\u30D7\u30EC\u30FC\u30C8
-
-pref.page.name.sql.completion         = SQL\u306E\u5B8C\u4E86
```
Comment (yoshinorin) [inline plugins/org.jkiss.dbeaver.ui.editors.sql/OSGI-INF/l10n/bundle_ja.properties:144]: The current translation is wrong.

File: plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/internal/UINavigatorMessages_ja.properties [hunk 4]
```diff
@@ -25,8 +25,13 @@ dialog_filter_remove_button =\u524A\u9664\u3059\u308B
 dialog_filter_save_label =\u4FDD\u5B58\u3055\u308C\u305F\u30D5\u30A3\u30EB\u30BF
 dialog_filter_name_label =\u540D\u524D
 dialog_filter_hint_text =\u30D5\u30A3\u30EB\u30BF\u3067\u306F\u30DE\u30B9\u30AF (%, _, *) \u3092\u4F7F\u7528\u3067\u304D\u307E\u3059
-
-actions_navigator_search_tip = \u30C6\u30FC\u30D6\u30EB\u540D\u306E\u4E00\u90E8\u3092\u3053\u3053\u306B\u5165\u529B
```
Comment (yoshinorin) [inline plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/internal/UINavigatorMessages_ja.properties:29]: The current translation is wrong.

File: plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/internal/UINavigatorMessages_ja.properties [hunk 5]
```diff
@@ -105,7 +130,7 @@ dialog_select_datasource_title =\u30C7\u30FC\u30BF\u30BD\u30FC\u30B9\u306E\u9078
 controls_object_list_job_props_read =\u30EC\u30A4\u30B8\u30FC\u30D7\u30ED\u30D1\u30C6\u30A3\u306E\u8AAD\u307F\u8FBC\u307F
 controls_object_list_message_items ={0}\u500B\u306E\u30A2\u30A4\u30C6\u30E0
 controls_object_list_message_no_items =\u30A2\u30A4\u30C6\u30E0\u306F\u3042\u308A\u307E\u305B\u3093
-controls_object_list_monitor_load_lazy_props =\u6020\u60F0\u306A\u30D7\u30ED\u30D1\u30C6\u30A3\u3092\u8AAD\u307F\u8FBC\u3080
```
Comment (yoshinorin) [inline plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/internal/UINavigatorMessages_ja.properties:108]: The current translation is wrong.

**Comment by E1izabeth** [comment]
Hello, @yoshinorin! Thank you for your work!

**Pull request merged**